src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/CanonicalizationDriver.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Xml;
+
+namespace CoreWCF.IdentityModel
+{
+  internal sealed class CanonicalizationDriver
+  {
```

Comment:
Incorrect indentation. Should be 4 spaces, not 2.

Reply:
done